recipes/supportdata/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "supportdata" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 606346ee39642a8395814b3a55bc9f9b8273414cc02311b45e4a44307272a51d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - filelock >=2.0
+    - pip
+    - python
+    - six >=1.5.2
+  run:
+    - filelock >=2.0
+    - python
+    - six >=1.5.2
+
+test:
+  imports:
+    - supportdata
```

Can we add ``pip check`` here as well, please?
```suggestion
    - supportdata
  requires:
    - pip
  commands:
    - pip check
```